runtime/doc/lua.txt

```diff
@@ -574,6 +574,15 @@ retained for the lifetime of a buffer but this is subject to change. A plugin
 should keep a reference to the parser object as long as it wants incremental
 updates.
 
+Required runtime files					*lua-treesitter-files*
+
+Treesitter requires some files to work properly :
+	- Parsers : this is the `.so` files created whan compiling parsers,
+	  they are searched under the `parsers` subdirectory.
+	- Queries : these are `.scm` files that are "sourced" when
+	  `vim.treesitter.get_query(lang, type)` is called. They are searched under the
+	  `queries/{lang}` subdirectory.
```

Comment:
This section should probably be a bit more formal.

Can you split it in into two different subsections: `tree-sitter-parsers` and `tree-sitter-queries` and give some info of both.

A simple example of a `query` would be good.

And also, do we ship with the `c` parser as a `parser.o` or something as well? We could give that as an example for the parser section.

Reply:
> And also, do we ship with the `c` parser as a `parser.o` or something as well? We could give that as an example for the parser section.

Hmm that's a good question, I think yes, but I will have to look at the runtime files.